library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -723,6 +729,10 @@ private Uri getDownloadUri(long id) {
         return ContentUris.withAppendedId(baseUri, id);
     }
 
+    public Uri getContentUri() {
```

Comment:
add this javadoc https://github.com/novoda/download-manager/pull/83/files#r34242943 back here